operator/metrics/metrics.go

```diff
@@ -138,6 +165,34 @@ func registerMetrics() []prometheus.Collector {
 	}, []string{LabelOutcome})
 	collectors = append(collectors, IdentityGCRuns)
 
+	CiliumEndpointBatchDensity = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: Namespace,
+		Name:      "number_of_ceps_per_ceb",
+		Help:      "The number of CEPs batched in a CEB",
+	}, []string{})
+	collectors = append(collectors, CiliumEndpointBatchDensity)
+
+	CiliumEndpointsChangeCount = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: Namespace,
+		Name:      "number_of_cep_changes_per_ceb",
+		Help:      "The number of changed CEPs in each CEB update",
+	}, []string{LabelOpcode})
+	collectors = append(collectors, CiliumEndpointsChangeCount)
+
+	CiliumEndpointBatchSyncErrors = prometheus.NewCounterVec(prometheus.CounterOpts{
```

Comment:
I don't think we need this, this is existing API request error with a filter of `resource=ciliumendpointbatches`.

Reply:
End users prefer having all metrics in single place. Nobody likes running into multiple places collect different metrics.

Reply:
I beg to disagree, see https://github.com/cilium/cilium/pull/17410#discussion_r713490823

Reply:
In Unmanaged kubernetes clusters, users to prefer to collect metrics from Prometheus

Reply:
Sure, but why is that related to this topic? you can config prom to scrape from apiserver...I'm confused, that's not a reason to keep redundant metrics :)

Reply:
It's not redundant metrics. If someone using Cilium prometheus scrape config to collect data, why they should depend on api-server data? This is part of Cilium feature, hence i would expect in Cilium exported from cilium Prometheus.

For reference, in Endpointslice implementation we do similar thing, nothing unusual here. https://github.com/kubernetes/kubernetes/blob/0ec2aaee473ef6256361bfa1aa044909dd0132fe/pkg/controller/endpointslice/endpointslice_controller.go#L551-L561